source/server-discovery-and-monitoring/server-monitoring.rst

```diff
@@ -158,6 +156,10 @@ be able to proceed anyway.
 Clients update the topology from each handshake
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+When a monitor check creates a new connection, it MUST perform the
+`connection handshake`_ and MUST use the handshake's ismaster response to
```

Comment:
nit: we can just say "MUST use the handshake's response to...", we've already begun the process of removing the term "master" from the server, and since we already call this thing a handshake there's no need to refer to `ismaster` here.

Reply:
Done.